### PR TITLE
Pfunc doc should reference Plazy for returning patterns

### PR DESCRIPTION
**Pull request by JordanHendersonMusic**
## Purpose and Motivation
Pfunc only works with values but the documentation does mention how to return a pattern, that is Plazy

## Types of changes

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

File: HelpSource/Classes/Pfunc.schelp [hunk 1]
```diff
@@ -4,7 +4,7 @@ categories:: Streams-Patterns-Events>Patterns>Function
 related:: Classes/Pfuncn, Classes/FuncStream
 
 description::
-Returns a link::Classes/Stream:: that returns values from the code::nextFunc::.
+Returns a link::Classes/Stream:: that returns values from the code::nextFunc::. To return a pattern, see Plazy.
```
Comment (cdbzb) [inline HelpSource/Classes/Pfunc.schelp]: should `Plazy` not be a link? link::Classes/Plazy

Comment (JordanHendersonMusic) [reply HelpSource/Classes/Pfunc.schelp]: Not too sure on how GitHub works, but I added another commit correcting this shortly after. It should be visible in the second commit. Is there something I have to do?

Comment (cdbzb) [reply HelpSource/Classes/Pfunc.schelp]: I see it - all good!

**Review by telephon — APPROVED** [review]
thanks!

**Pull request merged**